app/models/review.rb

```diff
@@ -0,0 +1,10 @@
+class Review < ActiveRecord::Base
+  belongs_to :bar
+  belongs_to :user
```

Comment:
If this `belongs_to :user`, there should be an update to the user model such that it `has_many :reviews`! Same with the `bar` model. It looks like you don't have a `bar` model yet either in this branch, so we'll need to make sure to merge in the Bar PR before this one so that the functionality isn't broken!